client/src/components/new-request/submit-request/submission-tabs.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <MainContainer>
     <template v-slot:container-header>
-      <v-col cols="auto" class="font-weight-bold h5 py-0 mt-2">
+      <v-col cols="auto" class="font-weight-bold h5 py-0">
```

Comment:
Example of what all header should look like:

![image](https://user-images.githubusercontent.com/10002889/101681695-68181680-3a17-11eb-9baf-6b910e911323.png)

In other words, vertically aligned and no extra top/bottom margins/padding (no top whitespace because it's the top element, and no bottom whitespace because it should not assume what comes after it).